backend/members/lookup.go

```diff
@@ -3,9 +3,24 @@ package members
 import (
 	"context"
 	"fmt"
+	"github.com/cloudevents/sdk-go/v2/event/datacodec/json"
 )
 
 // Lookup returns a member from the members api
 func (c *Client) Lookup(ctx context.Context, personID int) (*Member, error) {
 	return get[Member](ctx, c, fmt.Sprintf("persons/%d", personID))
 }
+
+func (c *Client) RetrieveByEmails(ctx context.Context, emails []string) (*[]Member, error) {
+	filter := struct {
+		Email map[string]any `json:"email"`
+	}{
```

Comment:
Personally I'm not a big fan of this kind of on the fly structs that are not empty. I'll say consider refactoring to a separate type (readability mostly).